etc/env/base/contracts.toml

```diff
@@ -38,9 +38,9 @@ L1_WETH_TOKEN_ADDR="0x5E6D086F5eC079ADFF4FB3774CDf3e8D6a34F7E9"
 L2_WETH_BRIDGE_ADDR="0x5E6D086F5eC079ADFF4FB3774CDf3e8D6a34F7E9"
 L2_WETH_TOKEN_IMPL_ADDR="0x5E6D086F5eC079ADFF4FB3774CDf3e8D6a34F7E9"
 L2_WETH_TOKEN_PROXY_ADDR="0x5E6D086F5eC079ADFF4FB3774CDf3e8D6a34F7E9"
-FRI_RECURSION_LEAF_LEVEL_VK_HASH ="0x72167c43a46cf38875b267d67716edc4563861364a3c03ab7aee73498421e828"
+FRI_RECURSION_LEAF_LEVEL_VK_HASH ="0x14628525c227822148e718ca1138acfc6d25e759e19452455d89f7f610c3dcb8"
 FRI_RECURSION_NODE_LEVEL_VK_HASH ="0x5a3ef282b21e12fe1f4438e5bb158fc5060b160559c5158c6389d62d9fe3d080"
```

Comment:
this didn't change?

Reply:
I do not know how you derive it, it leaf could change for many reasons, either because it captures base circuits VKs, or gate config changed for one if the base circuits and lead to new leaf circuit. But node is much simpler and should not be affected unless changed in it's internal logic